doc/whats_new.rst

```diff
@@ -44,6 +44,11 @@ Enhancements
 Bug fixes
 .........
 
+   - Fix a bug where :class:`sklearn.feature_selection.SelectFdr` did not 
+     exactly implement Benjamini-Hochberg procedure
```

Comment:
That's better, but indicating that it formerly may have selected fewer features than it should (and now does) would be more helpful for former users of this estimator.